flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/rules/common/WindowAggregateReduceFunctionsRule.scala

```diff
@@ -55,19 +58,12 @@ class WindowAggregateReduceFunctionsRule extends AggregateReduceFunctionsRule(
       newAgg))
   }
 
-  override def newCalcRel(
-      relBuilder: RelBuilder,
-      oldAgg: Aggregate,
-      exprs: util.List[RexNode]): Unit = {
-
-    // add all named properties of the window to the selection
-    val oldWindowAgg = oldAgg.asInstanceOf[LogicalWindowAggregate]
-    oldWindowAgg.getNamedProperties.foreach(np => exprs.add(relBuilder.field(np.name)))
-
-    // create a LogicalCalc that computes the complex aggregates and forwards the window properties
-    relBuilder.project(exprs, oldAgg.getRowType.getFieldNames)
+  override def newCalcRel(relBuilder: RelBuilder, rowType: RelDataType, exprs: util.List[RexNode]) {
```

Comment:
Please format the code again and add some inline comments to know what is happening here. Add `:Unit` as well.

Reply:
done